source/MaterialXCore/Definition.cpp

```diff
@@ -7,8 +7,8 @@
 
 #include <MaterialXCore/Document.h>
 
-namespace MaterialX
-{
+MATERIALX_NAMESPACE_BEGIN
+
```

Comment:
I'd recommend removing these double newlines after calls to MATERIALX_NAMESPACE_BEGIN, and setting the standard that only a single newline is needed after these calls.

Reply:
Done.